src/pages/composites/index.tsx

```diff
@@ -104,7 +104,7 @@ const orderBenefits: ServiceCardType[] = [
 	},
 ];
 
-const testimonials: TestimonialProps[] = [
+export const testimonials: TestimonialProps[] = [
```

Comment:
do you still need to export this type? Where is it being used? If it's not being used on this file, take it closer to where it's being used

Reply:
It's was used in InfoTesimonial.tsx but it has been moved into the component